modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenuSmoke.testcase

```diff
@@ -0,0 +1,303 @@
+				userEmailAddress = "userea@liferay.com",
+				userFirstName = "userfnA",
+				userLastName = "userlnA",
+				userScreenName = "usersnA");
+		}
+
+		task ("And signed out as guest user") {
+			SignOut.signOut();
+		}
+
+		task ("When the toggle shows underline to ON state") {
+			AccessibilityMenu.enableSpecificOption(key_toggleSwitchLabel = "Underlined Links");
+		}
+
+		task ("And sign in as a new user for first time") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfnA userlnA");
+		}
+
+		task ("And the toggle shows under to OFF state") {
+			AccessibilityMenu.disableSpecificOption(key_toggleSwitchLabel = "Underlined Links");
+		}
+
+		task ("And user signs out") {
+			SignOut.signOut();
+		}
+
+		task ("Then the toggle shows underline is set to ON") {
+			AccessibilityMenu.openAccessibilityMenu();
+
+			AssertChecked.assertCheckedNotVisible(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+		}
+
+		task ("And When guest user toggles show underline to OFF") {
+			Navigator.openURL();
+
+			AccessibilityMenu.disableSpecificOption(key_toggleSwitchLabel = "Underlined Links");
+		}
+
+		task ("And When guest user toggles show underline to ON") {
+			Navigator.openURL();
+
+			AccessibilityMenu.enableSpecificOption(key_toggleSwitchLabel = "Underlined Links");
+		}
+
+		task ("And sign in as user A") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfnA userlnA");
+		}
+
+		task ("Then show underline is set to OFF state") {
+			AccessibilityMenu.openAccessibilityMenu();
+
+			AssertNotChecked.assertNotCheckedNotVisible(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+		}
+	}
+
+	@description = "LPS-178192. Verifies that the user can enable it by Instance Settings."
+	@priority = 5
+	test CanBeEnabledByInstanceSettings {
+		task ("When Accessibility Menu is enabled at Instance Settings level") {
+			AccessibilityMenu.activateAccessibilityMenu(portlet = "Instance Settings");
+		}
+
+		task ("Then Accessibility Menu can be reached by keyboard") {
+			AccessibilityMenu.assertCanAccessAccessibilityMenuModal();
+		}
+	}
+
+	@description = "LPS-178192. Verifies that the user can enable it by Site Settings."
+	@priority = 3
+	test CanBeEnabledBySiteSettings {
+		property portal.acceptance = "false";
+
+		task ("When navigate to Site Settings > Accessibility And turn on Enable Accessibility Menu") {
+			AccessibilityMenu.activateAccessibilityMenuBySiteSettings(siteURLKey = "guest");
+		}
+
+		task ("Then Accessibility Menu can be reached by keyboard") {
+			AccessibilityMenu.assertCanAccessAccessibilityMenuModal();
+		}
+	}
+
+	@description = "LPS-178192.Verifies that the user can enable it by System Settings"
+	@priority = 5
+	test CanBeEnabledBySystemSettings {
+		property test.liferay.virtual.instance = "false";
```

Comment:
We don't need this property. Please let me know if you think otherwise.

Reply:
As we discussed on Slack, this property is necessary for these two tests.